src/components/AccountInfo.tsx

```diff
@@ -30,14 +31,15 @@ const AccountIcon = styled(Blockies)`
 export const Account = ({ account, open }: AccountProps) => {
   const theme = useTheme();
   const shortAccount = useMemo(() => centerEllipsis(account || ''), [account]);
+  const isMobileView = useMediaQuery(theme.breakpoints.down('md'));
 
   return (
     <Paper
       variant="outlined"
       sx={{
         display: 'flex',
         alignItems: 'center',
-        marginRight: theme.spacing(5),
+        marginRight: isMobileView ? '' : theme.spacing(5),
```

Comment:
`marginRight: { md: theme.spacing(5) }` 

I believe `marginRight: { md: 5 }`  should also be possible

Reply:
fixed